link-check/linkCompare.js

```diff
@@ -10,6 +10,7 @@ import { docx2md } from '@adobe/helix-docx2md';
  * @returns {boolean} - Returns true if the links have the same host and pathname, otherwise false.
  */
 export function compareLink(link1, link2) {
+  if (!link1 || !link2) return false;
```

Comment:
similar to above, feel that we should know that `link1` and `link2` are viable before the function call.

Reply:
Yeah, I think that makes sense here
```js
  const match = (link1 && link2) ? compareLink(link1, link2) : false;
```